Makefile

```diff
@@ -51,13 +51,16 @@ dev:
 	pdm install
 	if [ "$(CI)" != "true" ] && command -v pre-commit > /dev/null 2>&1; then pre-commit install; fi
 
+prerequisites:
+	pipx install --force mypy[reports]==1.10.0
+
 ########################################################################################
 # Lint and pre-commit
 ########################################################################################
 
 # Check lint with mypy.
 mypy:
-	pdm run python -m mypy .
+	mypy --python-executable $(shell pdm info --python) --html-report $(PUBLIC_DIR)/reports/mypy .
```

Comment:
The previous changes seems more consistent to use pdm to run mypy.😐

Reply:
Actually, I suppose to remove most of the `pdm run` here after having most of the dev-dependencies standalone.

Reply:
Understood.